vulcanai2/plotters/visualization.py

```diff
@@ -21,17 +20,56 @@
 import logging
 logger = logging.getLogger(__name__)
 
+def assert_display_available():
+    """
+    Check that display variable exists
+
+    Returns
+    -------
+    Boolean
+        True if display variable exists
+        False if display variable does not exist
+
+    """
+    return True if os.environ.get("DISPLAY") else False
```

Comment:
Why not just make this into a global variable? Not sure what is more pythonic but it's certainly simpler to go DISPLAY_AVAILABLE = os.environ.get("DISPLAY")